backend/app/api/endpoints/memory.py

```
@@ -0,0 +1,256 @@
+        updated_at = result.get("updated_at", result.get("updatedAt"))
+
+        return MemoryResponse(
+            id=memory_id,
+            content=request.content,
+            created_at=str(created_at) if created_at else None,
+            updated_at=str(updated_at) if updated_at else None,
+        )
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)